packages/docs/.vitepress/theme/components/VueMasteryBanner.vue

```diff
@@ -0,0 +1,291 @@
+<template>
+  <div class="vuemastery-banner-wrapper" role="banner" v-if="isVisible">
+    <div
+      :class="{ 'show-flash': showFlash }"
+      class="vuemastery-background-dim"
+      ref="vuemastery-banner-flash"
+    ></div>
+    <a id="vm-banner" href="https://www.vuemastery.com/holiday" target="_blank">
+      <img
+        id="vm-logo-full"
+        src="/vuemastery/vuemastery-white.svg"
+        alt="vuemastery"
+      />
+      <img
+        id="vm-logo-small"
+        src="https://firebasestorage.googleapis.com/v0/b/vue-mastery.appspot.com/o/flamelink%2Fmedia%2Fvue-mastery-logo-small.png?alt=media&token=941fcc3a-2b6f-40e9-b4c8-56b3890da108"
+        alt="vuemastery"
+      />
+      <div class="vm-banner-wrapper">
+        <div class="vm-banner-content">
+          <h1 class="vm-banner-title">Learn Vue with Evan You</h1>
+          <p class="vm-banner-sub">Get 60% off a year of Vue courses</p>
+        </div>
+        <button id="vm-banner-cta">Unlock your discount</button>
+      </div>
+      <button id="vm-banner-close" @click.prevent="closeBanner">
+        <span class="close">&times;</span>
+      </button>
+    </a>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+
+const isVisible = ref<Boolean>(true)
+const showFlash = ref<Boolean>(false)
```

Comment:
```suggestion
const isVisible = ref(true)
const showFlash = ref(false)
```

BTW it's `boolean` in TS. `Boolean` is the constructor. Here the type is just inferred.